### ISSUE TITLE
A11y_code.visualstudio.com_API Working With Extensions_Continuous Integration_Usability: Screen reader focus is going on decorative content i.e. Separator.

### ISSUE THREAD

**Issue by RohitJadhav-TCS** [microsoft/vscode-docs#3710]
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-manai%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458847260936&sdata=T26HQfSGOlnuRQdX%2ByXk%2B2bxqgwFvCIVfuboZUWidYY%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#A11y_Code.visualstudio.com_Benchmark;#A11ySev2;#A11yTCS;#A11yUsable;#Benchmark;#ChromiumEdge;#Chrome;#DesktopWeb;#Win10;#Visual Studio Code;#Narrator;#CAITestPass_May2020;#FTP;

**Environment Details**:
Application Name: code.visualstudio.com
#URL:https://code.visualstudio.com
Windows Version: Windows10
Browser version: Microsoft Edge Version 84.0.516.1 (Official build) dev (64-bit)
Screen reader: Narrator, NVDA Version 2020.1

**Repro Steps**:
1. Launch the application URL: https://code.visualstudio.com
2. Navigate to "API" link.
3. From left navigation, select "Working With Extensions".
4. Navigate to "Continuous Integration" link.
5. Turn on narrator and navigate to "Was this documentation helpful?" section at bottom and navigate with capslock+Left/Right arrow keys.

**Actual**:
Screen reader focus is going on decorative content i.e. separator.

This issue is reproducing only in ChromiumEdge+Narrator.
In Chrome+NVDA screen reader focus is not going on separator.
. 
**Similar issue observed in through out the application where "Was this document helpful?" section is present.**

**Expected**:
Screen reader focus should not go on decorative content. 

**Attachment link for Reference**:
https://microsoft-my.sharepoint.com/:v:/p/v-rojad/Efx04RR5WE1EnixVbIN-4k0BHP0fJCSJYVrlKxo9Oqy89A?e=lbLWBq

**Comment by JacksonKearl** [comment]
It's just a `<br/>` on our end, also unable to reproduce. Seems like a bug in the reader.
